src/content/app/genome-browser/components/zmenu/ZmenuAppLinks.tsx

```diff
@@ -35,6 +37,7 @@ type Props = {
 
 const ZmenuAppLinks = (props: Props) => {
   const genomeId = useSelector(getBrowserActiveGenomeId);
```

Comment:
Same here. `genomeId` can be imported on the line below.

Reply:
✅